tpm/aziot-tpm-sys/azure-iot-hsm-c/inc/hsm_client_data.h

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) Microsoft. All rights reserved.

```

Comment:
"HSM" is a misnomer since this lib is only for TPMs. That applies to all the files being added.

For this header specifically, why does this header need to exist? The `aziot_tpm_*` functions just forward to it, so you could merge them?

Reply:
Whoops, I thought I left a DEVNOTE somewhere that explains why I did that. Oh, yep, it's part of the 
63ec387 commit message. Copied here for reference:

> at the moment, this is simply a facade over the existing hsm_sys functions, in an effort to keep tests passing.

You're absolutely right - the two headers could be merged into one. It's moreso that I'd rather not spend time fixing the [one remaining] C test in-tree. We can re-visit whether or not this API should be consolidated as the PR gets closer to being merged.